src/App.js

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
+import { Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Home from './components/home';
+import Navbar from './components/navigation';
+import Quotes from './components/quotes';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/200790994-438b482a-978d-4b24-89ca-09fd2975921a.png)

- [ ] Your Built with section should have the technologies and languages which you have used in this project :+1: to make your readme more descriptive :+1: